docs/register-bitcoin-stake.md

```diff
@@ -697,7 +699,48 @@ The rewards are distributed as follows:
   finalized blocks to ensure that all eligible stakers receive their rewards
   based on the voting power and commission rates at the time of finalization.
 
-### 5.2. Rewards Withdrawal
+### 5.2. Costaking Rewards
+
+Costaking enables earning additional rewards by staking both Bitcoin and BABY
+tokens simultaneously. The Babylon address receiving BTC staking rewards and
+the address used for BABY delegations must be the same for costaking rewards
+to be earned.
+
+For costaking one must have both active BTC delegations (via finality providers)
+and BABY delegations (via validators). There is no limit to the number of BTC
+and BABY delegations one can have, and all active delegations contribute to
+the costaking rewards.
+
+Costaking rewards are calculated using a user score based on your combined
+stake amounts. The score is based on `min(active_btc_satoshis, active_baby_tokens / score_ratio)`.
+The `score_ratio` is a parameter defined in the `x/incentive` module that
+determines the relative weight of BABY tokens to Bitcoin in the scoring.
+
+**Algorithm**
+
+Each user's costaking reward is proportional to their score relative to all
+costakers:
+* `user_score = min(active_btc_satoshis, active_baby_tokens / score_ratio)`
+* `user_reward = total_costaking_rewards × (user_score / sum_of_all_scores)`
+
+The `score_ratio` is currently set to 20,000, meaning every 20,000 BABY staked
+makes 1 BTC eligible for costaking rewards. Only BTC staked to active finality
+providers counts toward costaking. All BABY delegations count regardless of
+validator status.
+
+For example, Alice stakes 6 BTC and 50,000 BABY, giving her a score of
+`min(6, 50,000/20,000) = 2.5`. Bob stakes 6 BTC and 150,000 BABY, giving him
+a score of `min(6, 150,000/20,000) = 6`. If the total costaking reward is
+10,000 BABY and they are the only costakers, Alice receives 2,941 BABY and Bob
+receives 7,059 BABY.
+
+**Reward flow:**
+* Rewards are collected and accumulated in the costaking reward pool
+* Rewards distributed based on user scores
+* Rewards automatically sent to your incentive gauge alongside regular BTC staking rewards
+* Withdraw all BTC-related rewards using `MsgWithdrawReward` with `type: "btc_staker"`
```

Comment:
I think we should be explicit here 
* Withdraw all Bitcoin and Costaker rewards using `MsgWithdrawReward` with `type: "btc_staker"`